### PR TITLE
Make Inspect.Algebra docs examples more formatter-compatible

### PR DESCRIPTION
**Pull request by amatalai**

### CODE REVIEW

File: lib/elixir/lib/inspect/algebra.ex [hunk 1]
```diff
@@ -324,18 +324,37 @@ defmodule Inspect.Algebra do
...
-      iex> doc = Inspect.Algebra.container_doc("[", Enum.to_list(1..5), "]",
-      ...>         %Inspect.Opts{limit: :infinity}, fn i, _opts -> to_string(i) end)
+      iex> doc =
+      ...>   Inspect.Algebra.container_doc(
+      ...>     "[",
+      ...>     Enum.to_list(1..5),
+      ...>     "]",
+      ...>     %Inspect.Opts{limit: :infinity},
+      ...>     fn i, _opts -> to_string(i) end
+      ...>   )
```
Comment (josevalim) [inline lib/elixir/lib/inspect/algebra.ex]: Can we break this one into variables instead?

File: lib/elixir/lib/inspect/algebra.ex [hunk 2]
```diff
@@ -324,18 +324,37 @@ defmodule Inspect.Algebra do
...
-      iex> doc = Inspect.Algebra.container_doc("[", Enum.to_list(1..5), "]",
-      ...>         %Inspect.Opts{limit: 3}, fn i, _opts -> to_string(i) end)
+      iex> doc =
+      ...>   Inspect.Algebra.container_doc(
+      ...>     "[",
+      ...>     Enum.to_list(1..5),
+      ...>     "]",
+      ...>     %Inspect.Opts{limit: 3},
+      ...>     fn i, _opts -> to_string(i) end
+      ...>   )
```
Comment (josevalim) [inline lib/elixir/lib/inspect/algebra.ex]: Can we break this one into variables instead?

File: lib/elixir/lib/inspect/algebra.ex [hunk 3]
```diff
@@ -324,18 +324,37 @@ defmodule Inspect.Algebra do
...
-      ...>         %Inspect.Opts{limit: 3}, fn i, _opts -> to_string(i) end, separator: "!")
+      iex> doc =
+      ...>   Inspect.Algebra.container_doc(
+      ...>     "[",
+      ...>     Enum.to_list(1..5),
+      ...>     "]",
+      ...>     %Inspect.Opts{limit: 3},
+      ...>     fn i, _opts -> to_string(i) end,
+      ...>     separator: "!"
+      ...>   )
```
Comment (josevalim) [inline lib/elixir/lib/inspect/algebra.ex]: Can we break this one into variables instead?

File: lib/elixir/lib/inspect/algebra.ex (hunk 1 above)
Comment (amatalai) [reply lib/elixir/lib/inspect/algebra.ex]: Sure, I'll update it in next few minutes

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**